README.md

```diff
@@ -2,28 +2,97 @@
   <img src="https://github.com/pyscf/pyscf-doc/blob/master/logo/pyscf-logo.png" height="80px"/>
 </div>
 
-PySCF documentation
+PySCF Documentation
 ===================
 
-Installation
-------------
+## Overview
 
-* Prerequisites
-    - [sphinx](https://pypi.org/project/Sphinx/)
-    
-    - [sphinxcontrib-bibtex](https://pypi.org/project/sphinxcontrib-bibtex/) v1.0.0
+The PySCF documentation uses `sphinx-doc` and several associated extensions to generate static `html` files for the website.
+To simplify the command line work, we use `make` to manage all building here, see [the Building section](#building) below or just run `make help` for more details.
 
-    - Set `PYTHONPATH` to include the PySCF source directory; otherwise, uncomment `sys.path.append(os.path.abspath('path_to_pyscf'))` in [source/conf.py](source/conf.py).
+## Building the Docs Locally
 
-* Make HTML pages
+### Requirements
 
-        make html
+Pip install the following packages:
 
-    The resulting html files can be found in the \"build/html\" directory.
+- pyscf
+- sphinx
+- sphinx-material
+- sphinxcontrib-bibtex
+- nbsphinx
 
-How to contribute
------------------
+> Pro-tip: Use `pyscf-doc/requirements.txt` to install all prerequisite packages at once.
+
+If you have multiple versions of PySCF on your machine and you would like so use a specific version, set `PYTHONPATH` to include the specific PySCF source directory you want; otherwise, uncomment `sys.path.append(os.path.abspath('path_to_pyscf'))` in [source/conf.py](source/conf.py).
+
+### Building
+All sphinx related sources files (i.e. `.rst` and `.md`) are contained in `source` and all webpage files (once they're generated) live in the `build`.
+
+To generate the website (without the API docs) run the following from the main project directory.
+If you are running on Linux, and you want to build faster you can use `make html_parallel`.
+
+```bash
+make html
+```
+
+To generate the complete website (including the API docs) run the following from the main project directory.
+Since the API docs are large, this build will noticeable slower than just generating the website with `make html`.
+:warning: PySCF must be accessible in your current Python environment when you run `make api_docs` or `make html_full`.
+
+```bash
+make html_full
+```
+
+To see more of the options you can use with `make`, just use `make help`.
+
+
+### Serving
+Finally to serve the website, you can run:
+
+```bash
+ python -m http.server --directory build/html
+```
+
+
+### Adding content to GitHub Pages
```

Comment:
Are we showing the website on github pages, too?

Reply:
pages should be small, no?